### ISSUE TITLE
We're missing a proper intervals file for GRCh37

### ISSUE THREAD

**Issue by maxulysse** [nf-core/sarek#693]
### Description of the bug

we have a `.list`, we should either have a `.bed` or a `.interval_list`

**Comment by giorgiagandolfi** [comment]
After renaming the interval file (.list) into .bed, I got this error:

Error executing process > 'NFCORE_SAREK:SAREK:PREPARE_INTERVALS:CREATE_INTERVALS_BED (wgs_calling_regions_Sarek.bed)'

Caused by:
  Process `NFCORE_SAREK:SAREK:PREPARE_INTERVALS:CREATE_INTERVALS_BED (wgs_calling_regions_Sarek.bed)` terminated with an error exit status (127)

Command executed:

  awk -vFS="	" '{
      t = $5  # runtime estimate
      if (t == "") {
          # no runtime estimate in this row, assume default value
          t = ($3 - $2) / 1000
      }
      if (name == "" || (chunk > 600 && (chunk + t) > longest * 1.05)) {
          # start a new chunk
          name = sprintf("%s_%d-%d.bed", $1, $2+1, $3)
          chunk = 0
          longest = 0
      }
      if (t > longest)
          longest = t
      chunk += t
      print $0 > name
  }' wgs_calling_regions_Sarek.bed
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_SAREK:SAREK:PREPARE_INTERVALS:CREATE_INTERVALS_BED":
      gawk: $(awk -Wversion | sed '1!d; s/.*Awk //; s/,.*//')
  END_VERSIONS

Command exit status:
  127

Command output:
  (empty)

Command error:
  WARNING: Skipping mount /var/singularity/mnt/session/etc/resolv.conf [files]: /etc/resolv.conf doesn't exist in container
  /bin/bash: line 0: cd: /beegfs/scratch/ric.cosr/ric.cosr/B1MG/work/7b/8c58dbb4a706cd1461d6f1daa88364: No such file or directory
  /bin/bash: .command.run: No such file or directory

Work dir:
  /beegfs/scratch/ric.cosr/ric.cosr/B1MG/work/7b/8c58dbb4a706cd1461d6f1daa88364

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

It sounded a little bit strange since both the work directory and the .command.run file are there.

**Comment by FriederikeHanssen** [comment]
This should be fixed in #727 . Let me know, if you run into any more issues :)

**Comment by diegogarcialopez** [comment]
I think I just faced this issue when running WGS data using GRCh37:

```
* --intervals (s3://ngi-igenomes/igenomes//Homo_sapiens/GATK/GRCh37/Annotation/intervals/wgs_calling_regions_Sarek.list): "s3://ngi-igenomes/igenomes//Homo_sapiens/GATK/GRCh37/Annotation/intervals/wgs_calling_regions_Sarek.list" does not match regular expression [\S+\.(bed|interval_list)$]
```
